File: src/wi_childcare_api/src/api/models/vaccination_provider.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize, Debug)]
pub struct VaccinationProvider {
    #[serde(rename(deserialize = "OBJECTID"))]
    id: i32,
    #[serde(rename(deserialize = "FACILITY_NAME"))]
    facility_name: String,
    #[serde(rename(deserialize = "ADDRESS"))]
    address: String,
    #[serde(rename(deserialize = "CITY"))]
    city: String,
    #[serde(rename(deserialize = "STATE"))]
    state: String,
    #[serde(rename(deserialize = "COUNTY"))]
    county: String,
    #[serde(rename(deserialize = "LAT"))]
    lat: f64,
    #[serde(rename(deserialize = "LON"))]
    long: f64,
    #[serde(rename(deserialize = "PHONE"))]
    provider_phone: String
}

File: src/wisconsin-dhs-dotnet/WisconsinDhs.Models/VaccinationProvider.cs
namespace WisconsinDhs.Models;

public class VaccinationProvider
{
    public int Id { get; set; }
    public string FacilityName { get; set; }
    public string Address { get; set; }
    public string City { get; set; }
    public string State { get; set; }
    public string County { get; set; }
    public float Latitude { get; set; }
    public float Longitude { get; set; }
    public string ProviderPhone { get; set; }
}